web_app/hazard-reporting-map/assets/icons/index.ts

```diff
@@ -3,5 +3,15 @@ import ChevronUp from './chevronUp.svg';
 import IncidentIcon from './incident.svg';
 import Users from './users.svg';
 import Location from './location.svg';
+import Location2 from './loation2.svg';
```

Comment:
Instead of using SVG for each icons, we can use something like font awesome.

Reply:
But that won't match with the icons provided in figma. I've used react icons before and was told to match the icons with figma.

Reply:
Got it.